UnitsNet.Tests/AssertEx.cs

```diff
@@ -7,9 +8,22 @@ namespace UnitsNet.Tests
     /// </summary>
     public static class AssertEx
     {
-        public static void EqualTolerance(double expected, double actual, double tolerance)
+        public static void EqualTolerance(double expected, double actual, double tolerance, ComparisonType comparisonType = ComparisonType.Relative)
         {
-            Assert.True(actual >= expected - tolerance && actual <= expected + tolerance, $"Values are not equal within tolerance: {tolerance}\nExpected: {expected}\nActual: {actual}\nDiff: {actual - expected:e}");
+            if(comparisonType == ComparisonType.Relative)
+            {
+                bool areEqual = UnitsNet.Comparison.EqualsRelative(expected, actual, tolerance);
+
+                double difference = Math.Abs(expected - actual);
+                double percentDifference = (difference / actual) * 100.0d;
+
+                Assert.True( areEqual, $"Values are not equal within relative tolerance: {tolerance * 100.0d}%\nExpected: {expected}\nActual: {actual}\nDiff: {percentDifference}%" );
```

Comment:
You can use format P2 as in `{0.0354321:P2}` to get `"3.54%"` string including the `%` symbol. It also uses the right symbol for the current culture.

Reply:
I would perhaps argue that percentage should be relative to `expected` and not `actual`, since you don't really know what actual is, but expected is .. well.. what you expect :-) If expected is `10` and actual is `0.000001`, then a difference of `~10` becomes a huge percentage instead of having an error of `~100%`.

Reply:
I didn't know you could do the P2 formatting outside of a ToString. Nice! I'll update that. And I agree that it should be against the expected.